docs/ref/request-response.txt

```diff
@@ -1213,9 +1213,10 @@ The :class:`StreamingHttpResponse` is not a subclass of :class:`HttpResponse`,
 because it features a slightly different API. However, it is almost identical,
 with the following notable differences:
 
-* It should be given an iterator that yields bytestrings as content. When
-  serving under WSGI, this should be a sync iterator. When serving under ASGI,
-  this is should an async iterator.
+* It should be given an iterator that yields bytestrings, :class:`memoryview`,
+  or string as content. Other types will be converted to a bytestring by
```

Comment:
I would suggest removing the added sentence about `Other types will be converted...` because I think it could lead the  reader to expect more of the class than what is offered. The tests do not consider general objects other than list of strings or list of bytes, so while I see what you mean in the `make_bytes` implementation, I would leave that undocumented as an implementation detail that could change in the future.

Reply:
I tend to agree with you, but I pretty much simply copied what was present for `HttpResponse`. Should we clean up what’s there for the `HttpResponse` as well then?
https://github.com/django/django/blob/ffff17d4b0117cce59f65c9f56fa164694eafd23/docs/ref/request-response.txt#L851

I’m happy just leaving that out for this part and not touch the other part as well.

Reply:
+1 to leave that part out and not touch the `HttpResponse` one. Let's keep the change simple and concise. Thanks!